src/index.js

```diff
@@ -245,8 +268,8 @@ class ReactTooltip extends Component {
     }
 
     this.setState({
-      placeholder,
-      isEmptyTip,
+      originTooltip: originTooltip,
+      isMultiline: isMultiline,
```

Comment:
Just a small tweak, everything else looks good

```suggestion
      originTooltip,
      isMultiline,
```